### DISCUSSION TITLE
How to force katex to display italic partial sign?

### DISCUSSION THREAD

**Discussion by Firestar-Reimu** [KaTeX/KaTeX#3656] [Q&A]
![image](https://user-images.githubusercontent.com/65082432/173309199-2dc950e8-33a7-4d6c-b1b6-bc4ffb4cf90f.png)

I found that katex display mathjax-like italic partial sign here.

But I remember there is a ugly, shortened upright version of katex partial. ~~(I searched it for an hour but failed)~~

OK, maybe like this: (https://katex.org/docs/supported.html)

![Screenshot_20220615_012241](https://user-images.githubusercontent.com/65082432/173638857-6512359d-a4a9-4ca3-ab22-64a3803aff50.png)

And this is the main reason I do not use katex.

How to force katex to display italic partial sign?

**Comment by edemaine** [comment]
Moved to #3658. (Please don't make multiple issues/discussions for the same thing.)
